src/lib/Tab/LocationView/ContentTab.php

```diff
@@ -124,9 +130,23 @@ public function loadContentLanguages(Content $content): array
             return $language->enabled && in_array($language->languageCode, $contentLanguages, true);
         };
 
-        return array_filter(
-             $this->languageService->loadLanguages(),
-             $filter
-         );
+        $languagesByCode = [];
+
+        foreach (array_filter($this->languageService->loadLanguages(), $filter) as $language) {
+            $languagesByCode[$language->languageCode] = $language;
+        }
+
+        $saLanguages = [];
+
+        foreach ($this->siteAccessLanguages as $languageCode) {
+            if (!isset($languagesByCode[$languageCode])) {
+                continue;
+            }
+
+            $saLanguages[] = $languagesByCode[$languageCode];
+            unset($languagesByCode[$languageCode]);
```

Comment:
Ok so we end up needing this several places, so guess you are in favour of us making this default behavior of `loadLanguages()` in siteAccess aware Repo layer?